stable/lamp/templates/deployment.yaml

```diff
@@ -362,7 +362,7 @@ spec:
       containers:
       {{- if .Values.php.fpmEnabled }}
       - name: "httpd"
-        image: "httpd:2.4.29-alpine"
+        image: "httpd:2.4-alpine"
```

Comment:
we should probably pull this image repo and version from a value so it can be pulled from a non public repo.